scripts/build.sh

```
@@ -9,29 +9,16 @@ npm install
-cd ../../..
-
-# Build FileSystem adapter
-cd packages/adapters/filesystem
-npm install
-npm run build
-cd ../../..
+# Build adapters
+for dir in packages/adapter-*/; do
+  if [ -d "$dir" ]; then
+    echo "Building $dir"
+    cd "$dir"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)